scripts/measurements/tools_run.sh

```diff
@@ -18,27 +19,23 @@ do
     #actual run with output of system time and max memory
     #/usr/bin/time  --format "$pcapFile,%S,%M" bro -r $pcapFile
     #actual run with output of elapsed time and max memory
-    /usr/bin/time  --format "$pcapFile,%e,%M" bro -r $pcapFile
-done >> performance.log 2>&1
-
-
-echo -e "\n--------------------------\n" >> performance.log
-echo -e "ntopng performance\n" >> performance.log
-for pcapFile in `ls Test_pcap_files/sample_capture_test*.pcap`
+    sudo /usr/bin/time  --format "$pcapFile,%e,%M" bro -r $pcapFile
```

Comment:
why do we need sudo here?

Reply:
Without sudo there was a warning message 
warning: can't stat directory .state/.tmp: Permission denied